### PR TITLE
Home task (Task3)  v2

### PR DESCRIPTION
**Pull request by AaronMqz**
this version is compatible with the main branch

### CODE REVIEW

File: netflix_roulette/src/components/Title/Title.component.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { Title } from "./Title.styled";
+
+export const TitleComponent = ({ children }) => {
+  return (
+    <>
+      <Title>{children}</Title>
+    </>
+  );
```
Comment (yhormanp-epam) [inline netflix_roulette/src/components/Title/Title.component.js:9]: Proptype must be used to validate the children sent and the type of data expected

**Review by yhormanp-epam — CHANGES_REQUESTED** [review]
Great implementation.
The proptypes were used correctly.
There is just one issue, one the components needs to specify the proptypes for the property expected.

Apart from that, everything well implemented

File: netflix_roulette/src/components/Title/Title.component.js (hunk 1 above)
Comment (AaronMqz) [reply netflix_roulette/src/components/Title/Title.component.js:10]: proptype was added

**Review by yhormanp-epam — APPROVED** [review]
All the acceptance criteria was correctly implemented. 

Excellent job

**Pull request merged**